loader/disksync.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015 The btcsuite developers
```

Comment:
this file should be renamed to something more meaningful, or move the function to another file.